package.json

```diff
@@ -61,7 +63,9 @@
     "formik": "^2.2.6",
     "js-cookie": "^2.2.0",
     "lodash": "^4.17.11",
+    "normalizr": "^3.6.1",
     "prop-types": "^15.7.2",
+    "ramda": "^0.27.1",
```

Comment:
are you sure it is good idea to use `ramda` together with `lodash`? this libraries have the same perpose. you need need to choose one. also if you want to use `lodash` utils in more functional programming style you can use `lodash/fp`

Reply:
removed ramda